docs/diginstroom/sip/2.0/profiles/bibliographic.md

```diff
@@ -0,0 +1,747 @@
+            <premis:relatedEventIdentifierType>UUID</premis:relatedEventIdentifierType>
+            <premis:relatedEventIdentifierValue>uuid-34ae79f8-a8e7-4768-a269-4d6d895662d6</premis:relatedEventIdentifierValue>
+        </premis:relatedEventIdentifier>
+    </premis:relationship>
+
+[...]
+
+</premis:premis>
+```
+
+<a id="example-premis-hassource"></a>_Example 5: a PREMIS `has source` relationship_
+
+```xml
+<premis:premis version="3.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:premis="http://www.loc.gov/premis/v3" xsi:schemaLocation="http://www.loc.gov/premis/v3 https://www.loc.gov/standards/premis/premis.xsd">
+    
+    [...]
+
+    <premis:relationship>
+        <premis:relationshipType authority="relationshipType" authorityURI="http://id.loc.gov/vocabulary/preservation/relationshipType" valueURI="http://id.loc.gov/vocabulary/preservation/relationshipType/der">derivation</premis:relationshipType>
+        <premis:relationshipSubType authority="relationshipSubType" authorityURI="http://id.loc.gov/vocabulary/preservation/relationshipSubType" valueURI="http://id.loc.gov/vocabulary/preservation/relationshipSubType/hss">has source</premis:relationshipSubType>
+        <premis:relatedObjectIdentifier>
+            <premis:relatedObjectIdentifierType>UUID</premis:relatedObjectIdentifierType>
+            <premis:relatedObjectIdentifierValue>uuid-1711cd43-19d2-4d89-9259-17443fc7d75f</premis:relatedObjectIdentifierValue>
+        </premis:relatedObjectIdentifier>
+        <premis:relatedEventIdentifier>
+            <premis:relatedEventIdentifierType>UUID</premis:relatedEventIdentifierType>
+            <premis:relatedEventIdentifierValue>uuid-34ae79f8-a8e7-4768-a269-4d6d895662d6</premis:relatedEventIdentifierValue>
+        </premis:relatedEventIdentifier>
+    </premis:relationship>
+    
+    [...]
+
+</premis:premis>
+```
+
+### Validation
+
+The XML files that are required by this profile can be validated using the following XML schema definitions:
+
+| File | Format | XML Schema |
+| `mets.xml` | METS v1.12.1 | [mets.xsd](https://www.loc.gov/standards/mets/mets.xsd) |
+| `premis.xml` | PREMIS v3.0 | [premis-v3-0.xsd](https://www.loc.gov/standards/premis/v3/premis-v3-0.xsd) |
+| `mods.xml` | MODS v3.7 | [mods-3-7.xsd](https://www.loc.gov/standards/mods/v3/mods-3-7.xsd) |
+| `dc.xml` (if no MODS) | Dublin Core (custom schema) | [dc_basic.xsd](https://raw.githubusercontent.com/viaacode/sipin-sip-validator/main/app/resources/xsd/dc_basic.xsd)<br>_depends on: [edtf.xsd](https://raw.githubusercontent.com/viaacode/sipin-sip-validator/main/app/resources/xsd/edtf.xsd), [dcterms.xsd](https://github.com/viaacode/sipin-sip-validator/blob/main/app/resources/xsd/dcterms.xsd), [dcmitype.xsd](https://raw.githubusercontent.com/viaacode/sipin-sip-validator/main/app/resources/xsd/dcmitype.xsd), [dc.xsd](https://raw.githubusercontent.com/viaacode/sipin-sip-validator/main/app/resources/xsd/dc.xsd)_ |
+
```

Comment:
This feels a bit lost here at the end, perhaps it should be provided higher up? Or we should provide a link to schemas on a separate page, or in the introduction?